requirements.txt

```diff
@@ -2,6 +2,6 @@ async_lru
 galaxy_importer
 jsonschema>=3.0
 packaging
-pulpcore>=3.7
+pulpcore>=3.7,<3.10
```

Comment:
is this okay? Or should we support 3.10?

Reply:
I think this is right. Plugins should plan to support the pulpcore current version + 1, so the current version is 3.8 so yes.